src/sentry/api/endpoints/project_group_index.py

```diff
@@ -291,7 +291,7 @@ def _subscribe_and_assign_issue(self, acting_user, group, result):
                 user=acting_user, key='self_assign_issue', default='0'
             )
             if self_assign_issue == '1' and not group.assignee_set.exists():
-                result['assignedTo'] = Actor(type=User, id=extract_lazy_object(acting_user).id)
+                result['assignedTo'] = Actor(type=User, id=acting_user.id)
```

Comment:
accessing .id automatically propagates the proxy object so no need for this explicit behavior